src/Components/GroupKeyword.php

```diff
@@ -122,9 +135,22 @@ public static function parse(Parser $parser, TokensList $list, array $options =
     public static function build($component, array $options = [])
     {
         if (is_array($component)) {
-            return implode(', ', $component);
+            $withRollup = false;
+            $expressions = array_map(static function ($component) use (&$withRollup) {
+                $withRollup = $component->withRollup;
+
+                return trim((string) $component->expr);
```

Comment:
```suggestion
                return trim((string) $component->expr, ' ');
```
So you only trim spaces, is that not better ?

Reply:
See new tests from [new commit](https://github.com/phpmyadmin/sql-parser/pull/417/commits/19ffba31018419a9cefa74838d63456faa18cf40)